Target-MVD-Mathias/ViewControllers/HomeViewController.swift

```diff
@@ -103,6 +106,21 @@ class HomeViewController: UIViewController {
     targetFormView.delegate = self
   }
   
+  private func getTargets() {
+    TargetAPI.targets(success: { targets in
+      self.targets = targets
+      
+      for target in targets {
+        self.addTargetToMap(target)
+      }
+      
+    }) { error in
+      self.showMessageError(errorMessage: error.domain, actionTitle: "Try again") { _ in
+        self.getTargets()
```

Comment:
This can make the user being involved in a loop until the request succeed. I think that it would be better just to show the error. WDYT?

Reply:
After discussing it with @pMalvasio I'm gonna just show the error and move the request to the viewWillAppear function so the user has the opportunity to fetch the targets every time he/she goes to the home screen